app/models/concerns/employee_activities.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module EmployeeActivities
+  extend ActiveSupport::Concern
+  include Capitalizable
+
+  included do
+    after_create :send_email_register
+    capitalize_attribute :first_name
+    capitalize_attribute :last_name
+
+    def self.create_with_account(params)
```

Comment:
Move it to services https://viblo.asia/p/rails-service-objects-guide-bWrZnNkmZxw